Smarty/templates/modules/Documents/pdfViewer.html

```diff
@@ -6,6 +6,7 @@
 
     <link rel="stylesheet" href="all.min.css">
     <link rel="stylesheet" href="viewer.css" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
```

Comment:
we don't use bootstrap, we use LDS. why do we need this?

Reply:
Hi @joebordes - I verified with @ochu-elhadji and they are used by the library itself.

I will to see if the features will work without these files otherwise I will download and make them to be internal files.